massa-client/src/cmds.rs

```diff
@@ -469,6 +469,22 @@ impl Command {
                 let amount = parameters[2].parse::<Amount>()?;
                 let fee = parameters[3].parse::<Amount>()?;
 
+                match amount.checked_add(fee) {
+                    Some(total) => {
+                        if let Ok(addresses_info) = client.public.get_addresses(vec![addr]).await {
+                            if !addresses_info.is_empty()
```

Comment:
I think if there is no address info we want to display something like "no balance information found for this address"

(can you use `get` here ? the `.is_empty` quite confused me at first)

Reply:
yep!

Reply:
I changed that :wink: